src/compiler/crystal/syntax/parser.cr

```diff
@@ -6253,6 +6252,16 @@ module Crystal
       @temp_arg_count += 1
       arg_name
     end
+
+    def consume_terminator
```

Comment:
I'm wondering if we could find a better name for this methor... Maybe `consume_expression_delimiter`?